edb/server/protocol/execute.pyx

```diff
@@ -255,8 +344,14 @@ async def execute_script(
         orig_state = state = dbv.serialize_state()
 
     data = None
+    recompile_requests = None
 
     try:
+        if debug.flags.func_cache:
+            if any(query_unit.user_schema for query_unit in unit_group):
+                # TODO(fantix): do this in the same transaction
+                recompile_requests = await dbv.clear_cache_keys(conn)
```

Comment:
We should still try to do recompiles with the text cache, right? We could drive it straight from the in-memory table, I think